pkg/workceptor/remote_work.go

```diff
@@ -90,7 +90,19 @@ func (rw *remoteUnit) GetConnection(ctx context.Context) (net.Conn, *bufio.Reade
 		rw.GetWorkceptor().nc.GetLogger().Info("Connection to %s failed with error: %s",
 			rw.Status().ExtraData.(*RemoteExtraData).RemoteNode, err)
 		errStr := err.Error()
-		if strings.Contains(errStr, "CRYPTO_ERROR") {
+		if strings.Contains(errStr, "CRYPTO_BUFFER_EXCEEDED") {
+			shouldExit := false
+			rw.UpdateFullStatus(func(status *StatusFileData) {
+				status.Detail = fmt.Sprintf("QUIC crypto buffer exceeded. CA bundle may be too large (limit: 16KB). See KCS 7129200: %s", errStr)
+				if !status.ExtraData.(*RemoteExtraData).RemoteStarted {
```

Comment:
Do we know that `status.ExtraData` is never `nil`? We may want to check that before attempting the type conversion. This will prevent a possible panic condition.

Reply:
You're right, that would be a different (and much larger) task. I want to be really clear: I'm definitely not asking for that.

My suggestion is only for the new instance you've added in this PR.

My reasoning is that the underlying field being asserted could potentially become nil between when the struct is initialized and when this assertion happens. What are your thoughts on adding that check just for the new code in this PR?